tests/e2e_tests/integration/00-setup.spec.ts

```diff
@@ -87,6 +87,12 @@ test.describe('Test setup', () => {
       ]);
       const page = await context.newPage();
       await page.goto(`${baseUrl}ui/#/settings`);
+      const isVisible = await page.isVisible(`text=/Change email/i`);
+      if (!isVisible) {
+        console.log('settings may not be loaded - move around');
+        await page.goto(`${baseUrl}ui/#/help`);
```

Comment:
Q: Why `help`, then `settings`?

Reply:
I suppose to trigger a refetch.

Reply:
So the `help` part could've been pretty much anything?

Reply:
the help pages are the most static area of the ui so I think no additional requests to the backend, navigating there is basically a UI only thing... afterwards the step to the settings gives the stripe request another chance to complete and show the settings - this should be just the fallback for the 700 timeout from above 🤞